app/controllers/api/databases.php

```diff
@@ -2669,7 +2669,8 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     });
 
 App::get('/v1/databases/:databaseId/collections/:collectionId/usage')
-    ->alias('/v1/database/collections/:collectionId/documents', ['databaseId' => 'default'])
+    ->alias('/v1/database/:collectionId/usage', ['databaseId' => 'default']) // endpoint pre-0.15.0
+    ->alias('/v1/database/collections/:collectionId/usage', ['databaseId' => 'default'])
```

Comment:
I put 2 aliases because `/v1/database/:collectionId/usage` was the endpoint pre-0.15.0, but I feel like it should have been `/v1/database/collections/:collectionId/usage`. Let me know if/how you want it changed.

Reply:
Yes, it should have been as you suggested, however as `alias` is only for backward compatibility I think we are safe with keeping it the way it was before.

Reply:
Updated per your recommendation.